cli/scripts/monorepo.ts

```diff
@@ -129,6 +129,17 @@ importers:
           }
         }
         this.commitFiles({ "yarn.lock": yarnYaml });
+
+        if (this.npmClient == "berry") {
+          execa.sync("yarn", ["set", "version", "stable"], {
```

Comment:
Is this really a desirable behavior?

Because for all projects with `yarn berry`, this will generate and set the yarn configuration along with a `.yarnrc.yml` and a `.yarn` folder.

If this is really desirable, it would be nice to put in the documentation with a warning.

Reply:
This file is for E2E it doesn't affect the end user.

Reply:
Oh, nvm